pkg/client/interfaces/types-prerecorded.go

```diff
@@ -8,7 +8,7 @@ package interfaces
 PreRecordedTranscriptionOptions contain all of the knobs and dials to control a Prerecorded transcription
 from the Deepgram API
 
-Please see the documentation for live/streaming for more details:
+Please see the live/streaming documentation for more details:
```

Comment:
Should say "see the prerecorded documentation"

Reply:
will update! another silly copy and paste error.